src/guide/frontend/ios.md

```diff
@@ -0,0 +1,134 @@
+---
+title: iOS Client Reference
+---
+
+# iOS Client Reference
+
+The purpose of this SDK is to be able to easily integrate [passkeys](https://developer.apple.com/passkeys/) into your iOS app with the help of the [Passwordless.dev](https://bitwarden.com/products/passwordless/) management system.
+
+## apple-app-site-association
+
+Apple requires associated domains to be set up to perform passkey registration and assertions. For more details on setting up an apple-app-site-association, see [Apple's documentation](https://developer.apple.com/documentation/Xcode/supporting-associated-domains).
+
+For passkeys, you need to set up a `webcredentials` section for your app.
+
+## Getting Started with the iOS SDK
+
+### Requirements
+
+* **Minimum iOS Version:** iOS 16
+* **Additional Capabilities:** 
+  * _Associate Domains_ are needed to add the domain of where the AASA file is hosted.
+
+### Swift Package Manager
+
+The SDK can be added to your project via [Swift Package Manager](https://www.swift.org/package-manager/). You can then access the SDK functionality using `import Passwordless`.
+
+### Initialization
+
+The SDK provides a `PasswordlessClient` object that needs to be initialized with a `PasswordlessConfig` configuration. The configuration contains the API URL and API key needed to interface with [Passwordless.dev](https://bitwarden.com/products/passwordless/). It also requires the relying party ID, which represents the domain of your server that hosts an apple-app-site-association file with your app's team ID and bundle ID.
+
+You will need to register with [Passwordless.dev](https://bitwarden.com/products/passwordless/) to obtain an API key.
+
+Here is an example of a configuration:
+
+```swift
+let passwordlessClient = PasswordlessClient(
+    config: PasswordlessConfig(
+        apiKey: "<YOUR API KEY>",
+        rpId: "demo.passwordless.dev"
+    )
+)
+```
+
+### Registration
+
+1. Request a registration token from your public RESTful API with the given username.
+2. Pass the registration token to the `PasswordlessClient` via the `register(token:)` function. This will prompt the user to use biometrics to create the local private key needed for validation and send the public key to the Passwordless.dev API. Once complete, a verification token will be returned. See [error section](#error-responses) for possible errors that could be thrown.
+
+```swift
+let verifyToken = try await passwordlessClient.register(token: registrationToken)
+```
+
+3. Pass the verification token to your public RESTful API to verify and return an authorization token. The user is now logged in to your app.
+
+![Registration Flow](./ios/Registration.gif)
+
+### Sign in
+
+There are two main approaches for passkey sign in: auto fill and manual entry.
```

Comment:
We should probably mention signInWithDiscoverable and signInWithUserId briefly, just to explain them, but we don't need to to guide the developer too much.